.env.sample

```diff
@@ -0,0 +1 @@
+LOAD_STORYBOOK=true
```

Comment:
It seems, that `LOAD_STORYBOOK` env variable must be present in all dotenvs? And should be `false` by default for the `.prod`